packages/cms/lib/modules/resource-form-widgets/lib/api.js

```diff
@@ -0,0 +1,180 @@
+module.exports = async function(self, options) {
+  /**
+   * Sync data with api when user saves or commit
+   * @param req
+   * @param doc
+   * @param options
+   * @returns {Promise<void>}
+   */
+  self.syncConfirmationFields = async (req, doc, options) => {
+    if ((doc.workflowLocale && doc.workflowLocale === 'default-draft') || (!doc.body || !doc.body.items)) {
```

Comment:
```suggestion
    if ((doc.workflowLocale && doc.workflowLocale === 'default-draft') || (!doc.body || !doc.body.items || !Array.isArray(doc.body.items))) {
```

Anders zou je `forEach` nog stuk kunnen gaan, toch?